includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -545,4 +332,91 @@ private function getHref( $hrefKey ) {
 	private function getSocialHrefs() {
 		return DesignSystemSharedLinks::getInstance()->getSocialHrefs( $this->lang );
 	}
+
+	private function getLinkTextObjectList( String $section, Array $linkKeys ) {
```

Comment:
What is our 'policy' around private functions - do we write unit tests for them?